manifests/package.pp

```diff
@@ -38,19 +38,22 @@
   if $elasticsearch::ensure == 'present' {
 
     # Create directory to place the package file
-    exec { 'create_package_dir_elasticsearch':
-      cwd     => '/',
-      path    => ['/usr/bin', '/bin'],
-      command => "mkdir -p ${elasticsearch::package_dir}",
-      creates => $elasticsearch::package_dir,
-    }
+    $package_dir = $elasticsearch::package_dir
```

Comment:
see below, its not entirely removed, its only guarded by a if $package_dir.
For OpenBSD, I've set it to undef, see in the params.pp.
Since I used the packages that come with the distribution, I don't have a need for the package_dir variable to be set. However, people should still be able to override it as parameter if really needed.